packages/eslint-plugin-mdx/src/rules/remark.ts

```diff
@@ -100,7 +100,7 @@ export const remark: Rule.RuleModule = {
             // related to https://github.com/eslint/eslint/issues/14198
             message: JSON.stringify(message),
             loc:
-              /* istanbul ignore next */ 'start' in place
+              /* istanbul ignore next */ place && 'start' in place
```

Comment:
How is happening? Is the value `undefined` expected here? Is that possible to add a test case for it?

cc @wooorm

Reply:
@JounQin Thank you so much for the hard work on the package. Your efforts are much appreciated.

I originally hit this problem while working with `https://github.com/JulianCataldo/remark-lint-frontmatter-schema` and thought it might just be that.

Turns out many of the `remark-lint` rules do not return a defined `place` variable.

I read through much of the vFile library to see if that **should** be defined.

[It appears that it can be undefined according to vfile.](https://github.com/vfile/vfile/blob/main/lib/index.js#L555C1-L556C1)

Reply:
Thanks for clarification.

I may have to enable `strictNullChecks` option later at https://github.com/mdx-js/eslint-mdx/blob/4623359cc9784d3e38bd917ed001c5d7d826f990/tsconfig.lib.json#L6

Reply:
Tracking at #522